apps/annotations/models.py

```diff
@@ -52,16 +54,7 @@ class Meta:
         ]
 
 
-class BaseTaggedModel(models.Model):
-    class Meta:
-        abstract = True
-
-    tags = TaggableManager(through=CustomTaggedItem)
-
-    def add_tags(self, tags: list[str], team: Team, added_by: CustomUser):
-        for tag in tags:
-            self.tags.add(tag, through_defaults={"team": team, "user": added_by})
-
+class AnnotationMixin:
     @property
```

Comment:
make this `cached_property` since it's called in a loop in the template

Reply:
https://github.com/dimagi/open-chat-studio/pull/344/commits/dfbc5fef42f3eef9e71b42152d7c9280c18d68f5